docs/web/examples/websocket/script.js

```diff
@@ -0,0 +1,24 @@
+function recordEvent(evtType, evt) {
```

Comment:
nitpick: can be have this as part of index.html ... just to reduce the number of files.

it should also make it clear that this is "webbrowser client-side" of the code and not some sort of nodejs backend code.

```suggestion
/**
 * This is an example of a webbrowser client-side websocket usage.
 */
function recordEvent(evtType, evt) {
```

Reply:
> nitpick: can be have this as part of index.html ... just to reduce the number of files.

I thought about doing this, but kept it separate in order to have better syntax highlighting and to break up the visual flow on the documentation page a bit.

Reply:
> it should also make it clear that this is "webbrowser client-side" of the code and not some sort of nodejs backend code.

Good point on this one, I will take that note.

Reply:
Hopefully the new file name addresses this.